### PR TITLE
Hey , I have added certain operation of Nlp.

### PR DESCRIPTION
**Pull request by Knlsharma**
Detailed meaning and theory can be found in my blog link.
Check readme .md!

### CODE REVIEW

File: core-nlp-example-master/pom.xml [hunk 1]
```diff
@@ -4,28 +4,41 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.techprimers.nlp</groupId>
-    <artifactId>core-nlp-example</artifactId>
+    <groupId>nlp</groupId>
```
Comment (MovingToWeb) [inline core-nlp-example-master/pom.xml]: can you please retain the same group id and artifact id?

**Review by MovingToWeb — COMMENTED** [review]
Hi Kunal,
Great to see first outside Contribution to TechPrimers. I will add you as collaborator in a while. meanwhile can you retain the existing files in the repo, so that my example in the Youtube video doesn't break with code?

**Comment by MovingToWeb** [comment]
I have integrated Travis CI with TechPrimers account, feel free to add the CI if it makes sense.

File: core-nlp-example-master/pom.xml (hunk 1 above)
Comment (Knlsharma) [reply core-nlp-example-master/pom.xml]: Sure.

**Comment by Knlsharma** [comment]
I will try my best because I have not used **Travis CI** much.

File: core-nlp-example-master/target/maven-archiver/pom.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
-#Mon Apr 29 16:14:38 IST 2019
+#Sat May 11 17:10:18 IST 2019
 version=1.0-SNAPSHOT
-groupId=nlp
-artifactId=exampl-pos
+groupId=com.techprimers.nlp
+artifactId=core-nlp-example
```
Comment (Knlsharma) [inline core-nlp-example-master/target/maven-archiver/pom.properties:5]: Hey , I've changed that !

Comment (MovingToWeb) [reply core-nlp-example-master/target/maven-archiver/pom.properties:5]: Cool. Can you bring back the CoreNlpExample.java? I will merge this once that file is added back

**Review by MovingToWeb — APPROVED** [review]

**Pull request merged**